Feature: pantry view

zacwalters4 opened the pull request:
This pull request will add the css and html needed to create the basic outline of the My Pantry view. There is a lot of room for stylistic improvement, but this pull request should unblock any functionality requiring this outline.

![Screen Shot 2022-11-03 at 3 07 25 PM](https://user-images.githubusercontent.com/22826695/199835030-2f8ef963-94b9-498f-abf1-46c01ee21aaa.png)

### In scripts.js
Add two functions `showPantry()` and `showFeaturedRecipe()` and placed these functions in their respective My and All Recipes view.

![Screen Shot 2022-11-03 at 3 10 01 PM](https://user-images.githubusercontent.com/22826695/199835168-9ce4b1e1-01bc-40d7-814a-d8996b8c1adc.png)

### In index.html
Add the pantry parent div and the following html elements under the banner section.

![Screen Shot 2022-11-03 at 3 09 39 PM](https://user-images.githubusercontent.com/22826695/199835143-5ef13b0f-1c79-4ed4-a623-89e00e0b51cf.png)

### In styles.css
Add styling for the created elements in index.html 

![Screen Shot 2022-11-03 at 3 11 01 PM](https://user-images.githubusercontent.com/22826695/199835214-b96d8d30-4976-4de8-9d83-4c4784ba180b.png)

![Screen Shot 2022-11-03 at 3 11 20 PM](https://user-images.githubusercontent.com/22826695/199835224-30082014-5b2e-46a3-97f7-bd07773aeced.png)

sam-rice commented:
Hey, this looks great! A couple thoughts:

- We might want to think about having `showPantry()` and `showFeaturedRecipe()` add/remove classes instead of adding inline style elements. I think our instructors have mentioned that this is the preferred method of dynamically changing CSS from scripts.js. Maybe we should bring this one to the team.
- Someone using a screen reader might infer that the entire page is a view of "My Pantry", since the heading added here is an `h1` element. If we use an `h2` instead, I believe that would imply that "My Pantry" is one of several pieces of content on this page.

zacwalters4 commented:
Changed both of those functions based on your feedback and also changed that h1 to an h2. I also changed some css styling to match the cleaner styling of the page a little more.

sam-rice commented:
Awesome! Thanks for making the tag change and styling update works really well with existing styling. Merging now.
